packages/node_modules/@webex/plugin-meetings/src/locus-Info/index.js

```diff
@@ -320,7 +320,12 @@ export default class LocusInfo {
   updateSelf(self) {
     if (self && !isEqual(this.self, self)) {
       this.updateMeeting(selfUtils.parse(self));
-      this.emitChanges(EVENTS.SELF_CHANGE, this.self, self);
+      if (selfUtils.isUnadmittedGuest(self)) {
+        this.emitChanges(EVENTS.SELF_UNADMITTED_GUEST, self);
```

Comment:
I dont think we need to send self as payload. Only the updated meeting object should be fine. We can discuss tomorrow